samples/Azure.Analytics.Purview.Account/Generated/PurviewAccountsClient.cs

```diff
@@ -1199,14 +1199,14 @@ internal HttpMessage CreateRegenerateAccessKeyRequest(RequestContent content, Re
             return message;
         }
 
-        internal HttpMessage CreateGetCollectionsRequest(string skipToken, RequestContext context)
+        internal HttpMessage CreateGetResourceSetRulesRequest(string skipToken, RequestContext context)
```

Comment:
This is just reordering of the methods due to reduced transform section in .md file